tools/st2ctl

```diff
@@ -122,6 +122,7 @@ function st2start(){
   else
     service mistral start
   fi
+  (cd /opt/stackstorm/static/webui && nohup python -m SimpleHTTPServer 80) &>> ${LOGFILE} &
```

Comment:
And yet we still run st2ctl with sudo every time (probably because of mistral service, not sure).

I would very much prefer us to run it on 80 port by default, but we can always provide env variable to change it. Deal?